.github/workflows/ci.yml

```diff
@@ -58,7 +58,7 @@ jobs:
 
       - name: Get changed files
         id: changed-files
-        uses: tj-actions/changed-files@ea251d4d2f03a9c18841ae1b752f58b82dfb4d5e # v35.3.0
+        uses: tj-actions/changed-files@487675b843e203b5c9a92a07f1ed763d046d7283 # v35.4.1
```

Comment:
This is actually quite impressive of renovate, but I suppose we don't want this update? Should this be reverted and pinned in the `renovate.json5` config?

Reply:
This is using a hash for security reasons and not to pin to a specific version.
https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions:~:text=Pin%20actions%20to%20a%20full%20length%20commit%20SHA
So I guess it's fine to update this unless there's a problematic change.